piptools/repositories/pypi.py

```diff
@@ -220,7 +220,7 @@ def resolve_reqs(self, download_dir, ireq, wheel_cache):
 
                 resolver = PipResolver(**resolver_kwargs)
                 require_hashes = False
-                if PIP_VERSION < (19, 4):
+                if PIP_VERSION < (20,):
```

Comment:
Maybe we shouldn't use `<` but `<=` to a version number that is already released.

Reply:
Do you mean this condition?
```python
if PIP_VERSION <= (19, 3):
    results = resolver._resolve_one(reqset, ireq, require_hashes)
else:
    resolver.require_hashes = require_hashes
    results = resolver._resolve_one(reqset, ireq)
```

Sounds good, but the condition would fail for the following minor releases, e.g. (19, 3, 1).

Reply:
Maybe `PIP_VERSION[:2] <= (19, 3)`?

Reply:
Yeah, that'll work! I'll try this in the following PR. Thanks! 👍